test/unit/pixelsDisplayed/index.spec.js

```diff
@@ -0,0 +1,251 @@
+/**
```

Comment:
Some small changes needed here to mock the the necessary items given use of global rather than arguments.